Makefile

```diff
@@ -463,6 +463,12 @@ release-only:
 		echo 'Please update Added: tags in the documentation first.' ; \
 		exit 1 ; \
 	fi
+	@if $(grep -IPqrs '^>>>>>>> [0-9A-Fa-f]+' benchmark deps doc lib src test tools) || \
```

Comment:
If that's the case, it's possible they could leave the `=======` behind too and we wouldn't have an easy way of checking for that.